lib/multiple_search_selection.dart

```diff
@@ -46,7 +46,7 @@ class MultipleSearchSelection<T> extends StatefulWidget {
     bool? showShowedItemsScrollbar,
     bool? showSelectAllButton,
     bool? showClearAllButton,
-    InputDecoration? searchFieldInputDecoration,
```

Comment:
Why have you remove this one ?

Reply:
Sorry, I don't think I meant to submit this deletion, I was experimenting with trying to get the clear text field button working and must have removed this then. I will fix...